m3u8/parser.py

```diff
@@ -375,6 +375,15 @@ def _cueout_envivio(line, state, prevline):
     else:
         return None
 
+def _cueout_duration(line):
```

Comment:
This doesn't seem to get called - you need to add it to `_parse_cueout`

Reply:
updated parser.py "_parse_cueout"